### DISCUSSION TITLE
Questions about training sets and potential functions

### DISCUSSION THREAD

**Discussion by Edinbur** [deepmodeling/dpgen#1498] [Q&A]
I have a little confusion when performing AIMD to prepare the initial training set.
Is it necessary to set the calculation precision higher when performing AIMD calculations? Is the force error related to the accuracy of the AIMD calculation? Because my potential function still has a relatively large force error after several **dpgen** iterations, around 0.2eV, while the energy error is relatively small, I would like to know how I can reduce the force error to a relatively low level?

**Comment by njzjz** [comment]
> Is the force error related to the accuracy of the AIMD calculation?

It's related according to the feedback in https://github.com/deepmodeling/deepmd-kit/discussions/2704

**Reply by Edinbur** [reply]
Thank you for your answer！
Am I able to elevate the truncation energy for single point energy calculations and increase the accuracy of the calculations in the 02.fp session of dpgen? So that new datasets can be added.
